cpt-dl/conda-recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
 package:
   name: cptdl
-  version: "1.1.5"
+  version: "1.1.6"
```

Comment:
You may have conflicts here because the cpt-dl package no longer exists. Resolve them by removing the conflicting file.